src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/PrisonerSearchByPrisonerNumbersResourceTest.kt

```diff
@@ -0,0 +1,151 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.resource
+
+import com.github.tomakehurst.wiremock.client.WireMock.*
+import org.awaitility.kotlin.await
+import org.awaitility.kotlin.matches
+import org.awaitility.kotlin.untilCallTo
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.web.reactive.function.BodyInserters
+import uk.gov.justice.digital.hmpps.prisonersearch.QueueIntegrationTest
+import uk.gov.justice.digital.hmpps.prisonersearch.services.PrisonerListCriteria
+import uk.gov.justice.digital.hmpps.prisonersearch.services.dto.OffenderBooking
+
+class PrisonerSearchByPrisonerNumbersResourceTest : QueueIntegrationTest() {
+
+  companion object {
+    var initialiseSearchData = true
+  }
+
+  data class IDs (val offenderNumber: String)
+
+  @BeforeEach
+  fun setup() {
+
+      if (initialiseSearchData) {
+        val prisonerNumbers = getTestPrisonerNumbers(12)
+        prisonMockServer.stubFor(get(urlEqualTo("/api/offenders/ids"))
+          .willReturn(aResponse()
+            .withHeader("Content-Type", "application/json")
+            .withHeader("Total-Records", prisonerNumbers.size.toString())
+            .withBody(gson.toJson( prisonerNumbers.map { IDs(it) }))))
+        prisonerNumbers.forEach {
+          prisonMockServer.stubFor(get(urlEqualTo("/api/offenders/$it"))
+          .willReturn(aResponse()
+            .withHeader("Content-Type", "application/json")
+            .withBody(getOffenderBookingJson(it))))
+        }
+
+      setupIndexes()
```

Comment:
Very minor, looks like formatting has gone a bit weird, I think we settled of 4 tabs size for Kotlin

Reply:
thanks - will set that up